sites/fast-website/src/app/data/framework.data.ts

```diff
@@ -7,37 +7,35 @@ export interface FrameworkContentPlacementData {
 export const frameworkContentPlacementData: FrameworkContentPlacementData[] = [
     {
         body:
-            "Angular's default binding syntax will always set properties on an element. This works well for rich data, like objects and arrays, and also works well for primitive values so long as the Custom Element author has mapped any exposed attributes to corresponding properties.",
-        header: "Angular",
-        headerSubscript: "8.2.14",
+            "FAST works flawlessly with both Aurelia 1 and Aurelia 2, with full integration into the binding engine and component model.",
+        header: "Aurelia",
+        headerSubscript: "1 & 2",
     },
     {
         body:
-            "DIO uses a runtime heuristic to determine if it should pass data to Custom Elements as either properties or attributes. If a property is already defined on the element instance, DIO will use properties, otherwise it will fallback to attributes.",
-        header: "DIO",
+            "Angular supports binding syntax for setting both attributes and properties, though the default binding syntax will always set properties on an element.",
+        header: "Angular",
+        headerSubscript: "8.2.14",
     },
     {
         body:
-            "hyperHTML will pass data to an element as properties, as long as the property is defined on the element's prototype. Otherwise it will fallback to passing data as attributes.",
-        header: "hyperHTML",
-        headerSubscript: "2.32.2",
+            "FAST works naturally with ASP.NET server-side development. Start building immediately by adding a script tag and using the custom HTML elements.",
+        header: "ASP.net",
```

Comment:
I think ASP.NET is stylized as all uppercase.

Reply:
Gah, I swear I saved before I pushed this :)